core/modules/field/field.module

```diff
@@ -1308,6 +1309,72 @@ function field_get_block_list() {
   return $types;
 }
 
+/**
+ * Helper function to retrieve the most commonly used label for a field.
+ *
+ * @param string $machine_name
+ *   The machine name for the field in question.
+ *
+ * @return string
+ *   The most commonly used machine name.
+ */
+function field_get_label($machine_name) {
+  if (!isset($machine_name)) {
```

Comment:
This is weird. Why would `$machine_name` ever be NULL? And if it just wasn't passed in, PHP will already throw an error. This seems like unnecessary input checking. (Though in PHP 7 we'll be able to simple type hint the parameter as always being a string).